deel-3-webpagina/styles/styles.css

```diff
@@ -1 +1,106 @@
-/*Schrijf hier jouw css*/
\ No newline at end of file
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap');
+
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+
+:root {
+    --blue:#151E3F;
+    --darkblue:#030027;
+    --beige:#F2F3d9;
+    --buff:#DC9E82;
+    --rose:#C16E70;
+}
+
+main {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin: 20px;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    background-color: var(--beige);
+    font-family: 'Roboto', sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    margin: 40px;
+}
+
+article {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    min-height: 350px;
+    text-align: center;
+    border-radius: 5px;
+    background-color: var(--rose);
+    flex-basis: 280px;
+    flex-shrink: 1;
+    flex-grow: 0;
+    margin: 10px 10px;
+}
+
+h1 {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+h3 {
+    display:flex;
+    flex-direction: row;
+    justify-content: center;
+    text-align: center;
+}
+
+
+p {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+
+}
+
+nav {
+    flex-basis: 60px;
+    background-color: var(--darkblue);
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 20px;
```

Comment:
Hier valt het me op dat je navigatiebalk niet doorloopt maar wit heeft rondom. Volgens mij is dit niet de bedoeling. Maar ik zie niet waar het aan ligt, bij mij was het opgelost toen ik de margin en padding in * op nul zette, maar dat heb jij ook gedaan.

Reply:
Ik had margin: 40px op de body, is er nu af :)